aten/src/ATen/native/cpu/SortingKernel.cpp

```diff
@@ -97,6 +149,10 @@ static void sort_kernel(
     // https://github.com/pytorch/pytorch/issues/91420
     return;
   }
+  if (can_use_radix_sort(values, descending)) {
+    parallel_sort1d_kernel(values, indices);
+    return;
+  }
```

Comment:
Hmm, we do support cpp17, right? If so, it is possible to re-use `std::sort` with the parallel policy...

Reply:
Though parallel execution policy was introduced some time ago, I don't think it is supported well throughout build tools. I made a quick experiment, with `execution::par` and `execution::par_unseq`, and I don't see any speedup (from profiling it seems that only a single core is employed anyway).